docs/references.bib

```diff
@@ -1 +1,316 @@
+ year         = {2024},
+ note         = {Accessed: 2024-11-18}
+}
+@article{scikit-learn,
+  title={Scikit-learn: Machine Learning in {P}ython},
+  author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.
+          and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.
+          and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and
+          Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},
+  journal={Journal of Machine Learning Research},
+  volume={12},
+  pages={2825--2830},
+  year={2011}
+}
+#
+# SCALING
+#
+
+# https://scikit-learn.org/stable/auto_examples/preprocessing/plot_scaling_importance.html
+@article{scikit-learn,
+  title={Scikit-learn: Machine Learning in {P}ython},
+  author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.
+          and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.
+          and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and
+          Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},
+  journal={Journal of Machine Learning Research},
+  volume={12},
+  pages={2825--2830},
+  year={2011}
+}
+@misc{scikit_learn_scaling_importance,
+ author       = {Scikit-learn Developers},
+ title        = {Importance of Feature Scaling},
+ howpublished = {\url{https://scikit-learn.org/stable/auto_examples/preprocessing/plot_scaling_importance.html}},
+ year         = {2024},
+ note         = {Accessed: 2024-11-18}
+}
+# https://www.baeldung.com/cs/normalization-vs-standardization
+@misc{baeldung_normalization_standardization,
+ author       = {A. Aylin Tokuç, Michal Aibin},
+ title        = {Normalization vs Standardization in Machine Learning},
+ howpublished = {\url{https://www.baeldung.com/cs/normalization-vs-standardization}},
+ year         = {2024},
+ note         = {Accessed: 2024-11-18}
+}
+#
+# TIME SERIES
+#
+
+# https://machinelearningmastery.com/autoregression-models-time-series-forecasting-python/
+@misc{machine_learning_mastery_autoregression,
+ author       = {Jason Brownlee},
+ title        = {Autoregression Models for Time Series Forecasting in Python},
+ howpublished = {\url{https://machinelearningmastery.com/autoregression-models-time-series-forecasting-python/}},
+ year         = {2021},
+ note         = {Accessed: 2024-11-18}
+}
+# https://machinelearningmastery.com/introduction-to-time-series-forecasting-with-python/
+@misc{machine_learning_mastery_time_series,
+ author       = {Jason Brownlee},
+ title        = {Introduction to Time Series Forecasting with Python},
+ howpublished = {\url{https://machinelearningmastery.com/introduction-to-time-series-forecasting-with-python/}},
+ year         = {2024},
+ note         = {Accessed: 2024-11-18}
+}
+
+#
+# STATIONARY DATA
+#
+
+# Forecasting: Principles and Practice (2nd ed)
+# Rob J Hyndman and George Athanasopoulos
+# 2018
+# https://otexts.com/fpp2/
+@book{hyndman_forecasting,
```

Comment:
for books, let's try to do chapter citations. if we need multiple sources (one for each chapter) that could be ok.